experiment/stop_experiment.py

```diff
@@ -29,8 +30,12 @@ def stop_experiment(experiment_name, experiment_config_filename):
     experiment_config = yaml_utils.read(experiment_config_filename)
 
     if experiment_config.get('local_experiment', False):
-        raise NotImplementedError(
-            'Local experiment stop logic is not implemented.')
+        new_process.execute(
+            ['make', 'stop-experiment', "EXPERIMENT=" + str(experiment_name)],
+            expect_zero=False)
```

Comment:
This is to avoid subprocess exception when the experiment normally stops since we don't make `make stop-expeirment` stop with no error when there is no docker to remove.